sdk/core/Azure.Core/src/Pipeline/DisposableHttpPipeline.cs

```diff
@@ -31,11 +31,11 @@ internal DisposableHttpPipeline(HttpPipelineTransport transport, int perCallInde
         }
 
         /// <summary>
-        /// Disposes the underlying transport if it is owned internally. If the underlying transport was created externally, nothing is disposed.
+        /// Disposes the underlying transport if it was created via the Build method on <see cref="HttpPipelineBuilder"/>. If the underlying transport was supplied as a custom transport on <see cref="ClientOptions"/>, nothing is disposed.
```

Comment:
```suggestion
        /// Disposes the underlying transport if it is owned by the client, i.e. it was created via the Build method on <see cref="HttpPipelineBuilder"/>. If the underlying transport is not owned by the client, i.e. it was supplied as a custom transport on <see cref="ClientOptions"/>, it will not be disposed.
```